content/en/docs/tasks/access-api-from-pod/access-api-from-pod.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Accessing the Kubernetes API from a Pod
```

Comment:
Hi @uhari03 . Thanks for creating this pull request.
Creating a new task page seems fine. I am wondering where the new page belongs in the task section?
Also, the task Markdown file and containing directory use the same name.
I suggest to place this new task into one of the existing task sub-folders or use a different name for the directory.

Reply:
@kbhawkey 
What do you think about moving the file under this directory: https://kubernetes.io/docs/tasks/access-application-cluster/

Reply:
@sftim, Since you authored the issue, what do you think about the current file location?

Reply:
>  What do you think about moving the file under this directory: https://kubernetes.io/docs/tasks/access-application-cluster/

That's not quite right either.

Eventually (separate to this issue) I'd like to see tasks tagged rather than have a focus on hierarchy. For now, hmm, how about moving just that one section inside https://kubernetes.io/docs/tasks/run-application/ ?

Reply:
👋 @uhari03
I'd like this PR to move forward.
What do you think about adding the new page under `/docs/tasks/run-application/`?

Reply:
Moved the new page to run-application

I also removed the description and added a weight of 120
The weight is most likely an outdated component, but just wanted to keep [it in line with the other files](https://github.com/kubernetes/website/pull/8602) under run application